RELEASE.md

```diff
@@ -56,9 +73,29 @@ priority.
 ### Post-Release Tasks
 
 #### Wait for evergreen
+
 Wait for the evergreen version to finish, and ensure that the release task completes successfully.
 
 #### Verify release artifacts
+
+Check that the released files, library and symbols, are available at the following URLs:
+- Windows
+  - Release build
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/windows/${release_version}/release/atsql.dll`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/windows/${release_version}/release/atsqls.dll`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/windows/${release_version}/release/atsql.pdb`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/windows/${release_version}/release/mongoodbc.msi`
+- Ubuntu 2204
+  - Release build
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/ubuntu2204/${release_version}/release/libatsql.so`
+    - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongosql-odbc-driver/ubuntu2204/${release_version}/release/mongoodbc.tar.gz`
+
+- Documentation
+  - `https://translators-connectors-releases.s3.us-east-1.amazonaws.com/eap/mongodb-odbc-driver/docs/MongoDB_ODBC_Guide.pdf`
+
+During the EAP, the following URLs are not used. They are retained in this document for legacy purposes, and will become correct again
```

Comment:
Do you think that's enough to flag this for when we will merge?
I'll try to keep an eye on it at that time, but if you have any other way to make this pop if we miss it when merging, I'll take it :)

Reply:
I don't, other than we know the urls will be invalid in the previous section!

Reply:
uhm, tricky.
Well, we will see and do our best to be extra careful when merging, to not merge this part except the documentation section.